### ISSUE TITLE
hpcstruct regression on develop branch

### ISSUE THREAD

**Issue by mxz297** [HPCToolkit/hpctoolkit#517]
hpcstruct running on a measurement directory seems broken. On ufront, I just built the develop branch and then run:

```
hpcrun /bin/ls
hpcstruct hpctoolkit-ls-measurements/
```

I saw the below errors and no struct files generated:

```
/bin/sh: -c: line 6: syntax error near unexpected token `('
/bin/sh: -c: line 6: `echo msg: begin parallel analysis of $cpubin_name \\(size = $nbytes, using 16 threads\\)'
make[1]: *** [Makefile:120: /home/xm13/hpctoolkit-gpu-samples/cu_vec_add/hpctoolkit-ls-measurements/structs/ls.hpcstruct] Error 1
make[1]: Target 'analyze' not remade because of errors.
/bin/sh: -c: line 6: syntax error near unexpected token `('
/bin/sh: -c: line 6: `echo msg: begin parallel analysis of $cpubin_name \\(size = $nbytes, using 1 threads\\)'
make[1]: *** [Makefile:120: /home/xm13/hpctoolkit-gpu-samples/cu_vec_add/hpctoolkit-ls-measurements/structs/ls.hpcstruct] Error 1
make[1]: Target 'analyze' not remade because of errors.
make: *** [Makefile:195: all] Error 2
ERROR: Make hpcstruct files for measurement directory failed
```

Running develop's hpcstruct on individual binaries seems to work fine.

The master branch does not have this problem.

**Comment by Jokeren** [comment]
@mxz297 I'm not able to reproduce this problem on ufront. Can you specify which commit of develop you were using?

**Comment by mxz297** [comment]
@blue42u and I briefly talked about it. There is a new script in the develop branch to handle `\` properly, so double `\` is no longer needed. Will be fixed by #497
